serverless.yml

```diff
@@ -8,7 +8,7 @@ plugins:
 provider:
   name: aws
   stage: dev
-  runtime: nodejs8.10
+  runtime: nodejs10.x
```

Comment:
Why not 12 or 14?

Reply:
because 10 worked and 16 didn't and I couldn't be bothered to try in between

Reply:
well node 16 doesn't exist on lambda so that was always a bit of a non starter...

12 or 14 have much faster cold start times so might help alleviate the problem where slack retries it a few times when it doesn't immediately respond and we get a few attempts at the emoji?

Reply:
14 doesn't seem to hang on response like 16 so have updated to that